src/entities/profile/setting-profile.entity.ts

```diff
@@ -0,0 +1,26 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { ProfileEntity } from './profile.entity';
+
+@Entity('setting')
+export class SettingEntity {
```

Comment:
I have wrote before, we have user table we don't need this table

Reply:
fixed